modules/tile-converter/src/i3s-converter/helpers/geometry-converter.ts

```diff
@@ -244,12 +244,13 @@ async function _makeNodeResources({
   const {faceRange, featureIds, positions, normals, colors, uvRegions, texCoords, featureCount} =
     generateAttributes(convertedAttributes);
 
-  if (tileContent.batchTableJson) {
-    makeFeatureIdsUnique(
+  let featureIdsMap: Record<string, number> = {};
+  if (propertyTable) {
+    featureIdsMap = makeFeatureIdsUnique(
```

Comment:
Some comment here maybe. Are we replacing original ids with unique ids or are we adding a new set of ids that are guaranteed to be unique?

I understand that it is often necessary / convenient to have unique ids, but when selecting etc, the user may expect the original ids.

Reply:
Done